packages/ember/tests/reexports_test.js

```diff
@@ -63,6 +63,12 @@ if (!jQueryDisabled) {
           assert.equal(body, document.body, 'Ember.$ exports working jQuery instance');
```

Comment:
This test is failing now. I think because the deprecation is now already triggered when accessing `Ember.$`, and not when calling it. Moving this [line](https://github.com/emberjs/ember.js/pull/17510/files#diff-fdceed71491044ac9b9895ffdc26c52bR60) into the `expectDeprecation()` callback should fix this!